modules/oe_theme_content_project/templates/pattern-field-list--variant-horizontal-oe-project.html.twig

```diff
@@ -0,0 +1,20 @@
+{#
```

Comment:
Can we use the field group template suggestions instead of creating one for the pattern? They should be already available to us.

Reply:
Fixed (made a commit to keep previous version)